spec/support/cohort.spec.js

```diff
@@ -0,0 +1,150 @@
+import Cohort from "../../src/cohort.js"
+describe('Cohort', () => {
+    let cohort 
+
+    beforeEach(() => {
+          cohort = new Cohort()
+    })
+
+    // Creating a cohort
+    it('creates a cohort with a name', () => {
+        const cohortName = 'Cohort 1'
+        cohort.createCohort(cohortName)
+
+        const result = cohort.cohortList
+        expect(result[0].name).toEqual(cohortName)
+    })
+
+    // Searching a cohort
```

Comment:
Tests look nicely organised! Something I've been doing myself instead of comments is having another describe block for each method, for example this would be:
```js
describe('searchCohort', () => {
// your various it's and different cases here
})
```